Makefile.install

```diff
@@ -99,7 +99,7 @@ INSTALLCMX = $(sort $(filter-out checker/% ide/% tools/% dev/% \
 	configure.cmx toplevel/coqtop_byte_bin.cmx plugins/extraction/big.cmx, \
 	$(MLFILES:.ml=.cmx)))
 
-install-devfiles:
+install-devfiles: $(GRAMMARCMA)
```

Comment:
Don't put it here, instead put it as a dependency of coqbinaries in makefile.build (I think that's the best place for this) (coqbinaries is a dependency of world)

Reply:
Makes sense. As recommended by @ejgallego I'll wait for input from other Make specialists before proceeding further though.

Reply:
Anyways the original dependency on this was a bug of the makefile as `CLexer` has never needed `grammar`. It is a regular camlp5 file.

Reply:
The thing is that the whole pre-reprocessing story of the makefiles is a bit "botched".

Reply:
The sanest option would be to migrate to Dune; the least insane option after that is imitate what Dune does and have the makefiles generate a `coqp5` binary to handle `ml4` processing.

Reply:
I agree with @SkySkimmer , install targets do not depend on the things they install... it is silly, but it is the way things are

Reply:
Change applied.